.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,28 @@
+name: Deploy docs
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 6 * * 1'  # Every Monday 6 AM UTC
+jobs:
+  build:
+    name: Deploy docs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download source
+        uses: actions/checkout@v2
+      - name: Install Crystal
+        uses: oprypin/install-crystal@v1
+        with:
+          crystal: nightly
+      - name: Install Python
+        uses: actions/setup-python@v2
+      - name: Install Python libs
+        run: pip install --no-deps -r requirements.txt
+      - name: Install Crystal libs
+        run: shards install
+      - name: Build site
+        run: mkdocs build
+      - name: Deploy to gh-pages
+        if: github.event_name == 'push' && github.ref == 'refs/heads/master'
```

Comment:
🤷, seems kinda redundant to build the docs if they're not going to be used for anything.  I can't say I had any problems with the other API doc deployments either.

Reply:
Eventually you'll resolve all warnings and will want to add a `--strict` flag. Then this will prevent new warnings from being added if someone makes a pull request against your doc.
Or prevent the doc generation from failing outright.

Reply:
👍 That's a fair point.